src/graphql/mirror.test.js

```diff
@@ -3627,6 +3689,79 @@ describe("graphql/mirror", () => {
     });
   });
 
+  describe("logging network requests", () => {
+    describe("_logRequest", () => {
+      it("saves query, query parameters, and request timestamp", async () => {
+        const db = new Database(":memory:");
+        const mirror = new Mirror(db, buildGithubSchema());
+        const query = [
+          Queries.build.query("MirrorUpdate", [], [Queries.build.field("foo")]),
+        ];
+        const queryParameters = {foo: "bar"};
+        const timestamp = new Date(123);
+        mirror._logRequest(query, queryParameters, timestamp);
+        const results = await db
+          .prepare(
+            "SELECT query, query_parameters, request_timestamp FROM network_log"
+          )
+          .get();
+        const expected = Queries.stringify.body(query, Queries.inlineLayout());
+        expect(results.query).toEqual(expected);
+        expect(results.query_parameters).toEqual(stringify({foo: "bar"}));
+        expect(results.request_timestamp).toEqual(
+          new Date(123).getMilliseconds()
```

Comment:
nit: I think it's ok to just use `123` here without going through a `Date` object.

Reply:
updated